core/templates/dev/head/components/follow_button/follow_button_directive.html

```diff
@@ -0,0 +1,21 @@
+<script type="text/ng-template" id="components/followButton">
+  <div class="oppia-footer-social">
+      <div>
```

Comment:
Deindent this and the following lines by 2. In HTML we always indent by exactly 2 when entering a child node.